src/scenedit/scen.sdfpicker.cpp

```diff
@@ -106,40 +106,43 @@ location cStuffDonePicker::run() {
 }
 
 void cStuffDonePicker::clamp_sdf() {
-	chosen_sdf.x = minmax(0, scenario.sdf_names.size() - 1, chosen_sdf.x);
-	chosen_sdf.y = minmax(0, scenario.sdf_names[0].size() - 1, chosen_sdf.y);
-	viewport.x = cols * floor(chosen_sdf.x / float(cols));
-	viewport.y = rows * floor(chosen_sdf.y / float(rows));
+	// Note: x and y in chosen_sdf are actually (r,c)
```

Comment:
This contradicts the other similar comments. Could that be the source of the crash you mentioned?